Source/ACE/Command/Handlers/AccountCommands.cs

```diff
@@ -11,19 +11,128 @@ namespace ACE.Command.Handlers
 {
     public static class AccountCommands
     {
-        // accountcreate username password
+        // accountcreate username password accesslevel
         [CommandHandler("accountcreate", AccessLevel.Admin, CommandHandlerFlag.ConsoleInvoke, 2)]
         public static void HandleAccountCreate(Session session, params string[] parameters)
         {
-            uint accountId = DatabaseManager.Authentication.GetMaxId() + 1;
-            string account  = parameters[0].ToLower();
-            string salt     = SHA2.Hash(SHA2Type.SHA256, Path.GetRandomFileName());
-            string password = SHA2.Hash(SHA2Type.SHA256, parameters[1]);
-            Account acc = new Account(accountId, account, salt, password);
+            uint accountId          = DatabaseManager.Authentication.GetMaxId() + 1;
+            string account          = parameters[0].ToLower();
+            string salt             = SHA2.Hash(SHA2Type.SHA256, Path.GetRandomFileName());
+            string password         = SHA2.Hash(SHA2Type.SHA256, parameters[1]);
+            AccessLevel accessLevel = AccessLevel.Player;
+
+            if (parameters.Length > 2)
+            {
+                switch (parameters[2].ToLower())
```

Comment:
You should be able to force an enum.Parse on these instead of essentially duplicating the list.  Would simplify the code a lot.